apps/bluetooth/locales/bluetooth.en-US.properties

```diff
@@ -70,7 +70,10 @@ active-pair-passkey      = To pair with {{device}}, enter the passkey on the dev
 passive-pair              = Bluetooth pairing request
 passive-pair-confirmation = {{device}} would like to pair with this phone. To accept, make sure that both devices currently have the following PIN displayed:
 passive-pair-pincode      = {{device}} would like to pair with this phone. To accept, enter the PIN on the device.
```

Comment:
We cannot remove it since it still services for the older Bluetooth_v1 code base.